src/types/types.ts

```diff
@@ -884,4 +884,117 @@ export interface PoiType {
   contents?: PoiType[]
 }
 
-export type PoiHiearachy = PoiType[]
+export type PoiHierarchy = PoiType[]
+
+
+export interface Location extends LatLngId {
+  properties?: {
+    /** Parameter used for gravitational criteria to set a custom attraction strength for the given location
+     * @default 1
+     */
+    gravitationAttractionStrength: number
```

Comment:
This should be optional instead. Passing an empty object `{}` should still succeed.